exporter/exporter.go

```diff
@@ -489,3 +503,32 @@ func (exporter *NebulaExporter) buildMetricMap(
 	}
 	return exporter
 }
+
+func convertToMap(metrics []string) map[string]string {
+	matches := make(map[string]string)
+	for t := 0; ; t += 2 {
+		if t == len(metrics) {
```

Comment:
if len(metrics) is Odd number,  metrics[t+1] will panic. 
`t + 1 >= len(metrics)` will be more safty.

Reply:
len(metrics) is even number， but t+1 may  avoid panic and more safety